README.md

```diff
@@ -469,35 +465,43 @@ Fully customisable.  |
 :star2: | Name | Description | 🌍
 --- | --- | --- | ---
 2 | [@mkatircioglu](https://github.com/mkatircioglu)/[**dotfiles**](https://github.com/mkatircioglu/dotfiles) |  |
-### Crystal #
```

Comment:
@lacivert Whoups! Because these were not added in the `package.json` file. Can you create a pull request adding these in JSON format in `package.json`? Otherwise I'll do it.

Thanks for heads-up!